lib/redis/connection/ruby.rb

```diff
@@ -2,6 +2,7 @@
 require "redis/connection/command_helper"
 require "redis/errors"
 require "socket"
+require "openssl"
```

Comment:
Of anything I could foresee actually causing problems it's this. How do you feel about a hard dependency on the OpenSSL extension? It's generally part of the standard library but I could see this causing problems for people who disable it...